types/block.go

```diff
@@ -1065,12 +1069,15 @@ func (data *Data) Hash() tmbytes.HexBytes {
 }
 
 // ComputeShares splits block data into shares of an original data square and
-// returns them along with an amount of non-redundant shares. The shares
-// returned are padded to complete a square size that is a power of two. The
-// size of the square is computed and recorded in data
-func (data *Data) ComputeShares() (NamespacedShares, int) {
-	// TODO(ismail): splitting into shares should depend on the block size and layout
-	// see: https://github.com/celestiaorg/celestia-specs/blob/master/specs/block_proposer.md#laying-out-transactions-and-messages
+// returns them along with an amount of non-redundant shares. If a square size
+// of 0 is passed, then the minimum square size of 1 is used. The size of the
+// square is computed and recorded in data
```

Comment:
I think what is still mildly confusing is the "The size of the square is computed and recorded in data" part. What does that mean?
Also: add punctuation.

Reply:
I can definitely see how this is confusing. I think I was referring to the local var 'data' that the method is acting on, so I made that clearer and then also changed the description to be more accurate in the case that 0 is passed.